File: chapter_3/golang/greet/Greet.go
package main

import "fmt"

func greet2(name string) {
	fmt.Printf("How are you, %s\n", name)
}

func bye() {
	fmt.Print("ok bye!")
}

func greet(name string) {
	fmt.Printf("Hello, %s!\n", name)
	greet2(name)
	fmt.Println("getting ready to say bye...")
	bye()
}

func main() {
	greet("Renan")
}

File: chapter_3/typescript/greet.ts
const greet2 = (name: string) => console.log(`How are you ${name}?`);

const bye = () => console.log("Ok, bye!");

const greet = (name: string) => {
  console.log(`Hello, ${name}!`);
  greet2(name);
  console.log("getting ready to say bye...");
  bye();
};

greet("Renan");
